app/controllers/edit.js

```diff
@@ -54,7 +54,5 @@ export default class EditController extends Controller {
     documentContainer.currentVersion = editorDocument;
     yield documentContainer.save();
   }
-  toggleDeleteModal() {
-    console.log('delete');
-  }
+  toggleDeleteModal() {}
```

Comment:
I don't remember what is this for? maybe we should remove the function or make it do something, I remember I just put the console.log in order to do something with it but forgot.

Reply:
I removed it for now, we can always add a delete option on the edit page later.